docs/api/progress.md

```diff
@@ -42,6 +42,7 @@ Content-Type: application/json
                 "audioUrl": "http://example.com/audio.mp3",
                 "recordingSet": {
                   "id": "5600160f578c4db6b7b0327e16e42153",
+                  "audioUrl": "http://example.com/fullaudio.mp3",
```

Comment:
you don't want to call it fullAudioUrl? Guess it's obvious from the structure, right?

Reply:
Fine by me.

Reply:
👍